e2e/next-ai-kitchen-sink/app/todo/page.tsx

```diff
@@ -271,3 +277,27 @@ export default function Page() {
     </main>
   );
 }
+
+function AiChatEmptyComponent({ chatId }: { chatId: string }) {
```

Comment:
We should export `AiChatComponentsEmptyProps` (and `AiChatComponentsLoadingProps` too even if it's empty for now I think) publicly from `"@liveblocks/react-ui"` and use it here.

Reply:
Good point. Also added JSDoc comments to the props.